config/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "Node.js Homework Application"
+}
```

Comment:
While having no empty line at the end is not a big issue, but I suggest to take a look on editorconfig & set up it for you. Also you can check prettier plugin to make the source code indentation & etc. automatically on save.